### PR TITLE
Add in light branding for the embedded map.

### LINKED ISSUE #570: Text Change for give.html [FTM global]
Please change text under embed map on give.html to:

**"In collaboration with [#findthemasks]. Don't see your healthcare facility listed? Fill in our [PPE Request Form] to join the map!"**

**[#findthemasks]** will link to the country-specific homepage that corresponds to the respective give.html page:
give.html links www.findthemasks.com
/ca/give.html links www.findthemasks.ca
/fr/give.html links www.findthemasks.com/fr _etc_.
**[PPE Request Form**] can remain the button to the respective country's request Form.

**We no longer need to stay unbranded**. In fact we need some marketing/branding assistance and with increased embedding from our existing partnerships and collaborators, this text-change is necessary. People reach out to us to discuss how to collaborate on the "GetUsPPE map". We all worked hard to make this global PPE needs map. Let's let the world know about Find the Masks!

### PR DESCRIPTION
**Pull request by awong-dev**
fixes #570

### CODE REVIEW

**Comment by linderr** [comment]
Mmmm.... did you mean to put this in give.html instead of "ftm-add-donation-site-question" of the main page? :)
image

**Comment by linderr** [comment]
The give pages work perfect though! I tried give.html, ca/give.html, fr/give.html --- they all link back to their respective homepage --- good job!

image

**Comment by awong-dev** [comment]
Ghaaa... good catch. I hacked in a fix for the english pages.  @nburt I don't know if I need to fill in all the other locales...or if I do, how I'm supposed to do that.  Punt!

**Comment by nburt** [comment]
trying to figure out why smartling is not picking up new changes... going to mess with this. Re button texts I think this should happen separately and it should be a global change. We're using the default bootstrap styles but we can apply our own theme (i.e. colors, fonts, etc.). If we need this, I am ok with a quick fix but I'd rather tackle theming with the larger ux overhaul if possible

**Comment by nburt** [comment]
Fixed translations (I was playing with settings in smartling yesterday to try and figure out how to get i18n working on the node side...)

My laptop is gonna die in a few minutes, will leave button color up to you two but things look good to me

File: public/i18n/en.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
   "ftm-ack-data-sources": "This site also includes data from <a href=\"https://tinyurl.com/hospitals-needing-ppe\">https://tinyurl.com/hospitals-needing-ppe</a>, a project by:",
   "ftm-add-donation-site-answer": "<a class='add-donation-site-form'><button style='margin-top:20px;' type='button' class='btn-sm btn-primary'>Complete this form</button></a>",
   "ftm-add-donation-site-header": "Do You Need PPE?",
-  "ftm-add-donation-site-question": "Do you work in a hospital or clinic and need PPE? Want to update or remove information?",
+  "ftm-collaboration-with": "Made in collaboration with <a href=\".\" target=\"_blank\">#findthemasks</a>.",
+  "ftm-add-donation-site-question": "Want to join the #findthemasks map? Need to update or remove info?",
```
Comment (linderr) [inline public/i18n/en.json:21]: Yes! Perfect.

**Comment by linderr** [comment]
@nburt yes, we can leave the button colour for larger UX style changes. @sumitsekhri @tina (what is your GH username??)

**Pull request merged**